perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -54,6 +80,107 @@
                   'num_machines': 1}
 
 
+JOB_FILE_TEMPLATE = """
```

Comment:
@yuyantingzero @noahl 

I am curious to know why using a job file template here makes it a "completely new benchmark". It's still FIO benchmark. :)

Perhaps it's because we are no longer testing directly against device if we use the template? but we can still specify the filename to something like '/dev/sdb' to test directly against device, right? 

Let me know if I am missing something here...